nautobot/extras/api/serializers.py

```diff
@@ -631,7 +631,7 @@ def validate(self, data):
         """
         Add the originating Request as a parameter to be passed when creating/updating a GitRepository.
         """
-        data["request"] = self.context["request"]
+        data["user"] = self.context["request"].user
```

Comment:
I don't think so, since one is generated by the change context if not provided. We're trying to get rid of any reason to use or pass requests around entirely.

Reply:
Does this pose problems for chained jobs wanting to use a common request id to correlate changelogs together?

Reply:
I believe celery uses a `GroupResult` [model](https://github.com/celery/django-celery-results/blob/28d9101843c146d66a7be35815fbc7a4acc8cb5c/django_celery_results/models.py#L175) to track chains of tasks. We might need to implement that.